README.md

```diff
@@ -71,25 +70,21 @@
 
 ### Key Features <a name="key-features"></a>
 
-- **[Displays List of ToDo Tasks]**
-- **[Addition of New Tasks]**
-- **[Removal of a Task from the list]**
+- **[Displays List of Players and Scores]**
+- **[Addition of New Players and Scores]**
+- **[Removal of all Players and Scores from the list]**
 - **[Local storage]**
-- **[Editing a task with Edit Button]**
-- **[Checkboxes are there to check the completed tasks]**
-- **[All the completed tasks can be Cleared by Clicking the Clear All Completed Button]**
-- **[A Reset button is also there to reset all the tasks.]**
+- **[A Reset button is also there to reset all the Players and Scores.]**
 - **[Webpack is managing all the dependencies and auto loading in browser]**
 - **[All the scetions and functionality is re-created by ES6 Modules]**
-- **[ToDo List is Interactive, checking and unchecking the checkbox is interactive.]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- <a href="https://mike111222.github.io/ToDoList/dist/">My ToDoList Project's Live Demo Link</a>
+- <a href="">Coming Soon</a>
```

Comment:
Since the project is not yet deployed, please let's remove the broken link for a professional repository. 🛠👍

![image](https://github.com/Mike111222/MyLeaderBoard/assets/78251927/7af4ce9d-e77d-4bf0-9faa-7286abec2a5b)

Reply:
I have made the changes in light of your kind suggestions. Thanks